manifests/qualys_eus_reporting.pp

```diff
@@ -1,56 +1,27 @@
-# @summary Add wrapper for subscription-manager to lie about enabled repos (requires that qualys can sudo to root)
-#   See SVC-12072 for details
-#
-# @param enabled
-#   Boolean to enable/disable wrapper for subscription-manager
-#
-# @param repos
-#   Array of hashes containing repos to add to the qualys_eus_reporting.sh script
+# @summary Remove old, historical wrapper for subscription-manager to lie about enabled repos.
+#   This only cleans up old files that are no longer needed.
+#   See SVCPLAN-5381 & SVC-12072 for historical details.
```

Comment:
I think we can add one extra statement, something along the lines of:
"Once all NCSA hosts have removed these files this class can go away"

Mainly looking for something to give this class a condition after which it can be fully deleted. I don't really have a preference where that is, either a comment here like I suggested, a ticket in jira, or an issue in this repo. Just something that can let someone unfamiliar with this know when it's safe to fully delete this class. Otherwise this class might just existing infinitely. Which admittedly isn't a huge deal but best to clean up cruft when we can.

Reply:
Good idea. That comment has been added now.